libzhlgen/test/functions/mac/1.6.13/ConfirmWindow.zhl

```diff
@@ -0,0 +1,27 @@
+"554889e553504889fb488bbba0000000":
+cleanup __amd64 void ConfirmWindow::Close(ConfirmWindow *this);
+"554889e5807f0800":
+cleanup __amd64 void ConfirmWindow::MouseClick(ConfirmWindow *this, int mX, int mY);
+".!554889e553504889fbe8????????488d05????????48890348c7432000000000":
+noHook __amd64 void AchievementTracker::DO_NOT_HOOK_1();
+"!554889e553504889fbe8????????488d05????????48890348c7432000000000":
```

Comment:
Previous signature was a `noHook` but this signature is a global match and not a relative (`.`) match, noHook might be unnecessary or this signature might be defined incorrectly.